contrib/check-config.sh

```diff
@@ -215,6 +223,10 @@ fi
 if [ "$kernelMajor" -lt 5 ] || [ "$kernelMajor" -eq 5 -a "$kernelMinor" -le 2 ]; then
 	check_flags NF_NAT_NEEDED
 fi
+# check availability of BPF_CGROUP_DEVICE support
```

Comment:
Is this intended to check `BPF_CGROUP_DEVICE` or `CGROUP_BPF`? :sweat_smile:

(From what I can tell, `CGROUP_BPF` is the config option that enables the underlying `BPF_CGROUP_DEVICE` functionality in the kernel, so probably splitting hairs a little here.)

Reply:
I was following along from the description in https://github.com/moby/moby/issues/41874#issue-784953371 .
Do you think this is a better comment ` "checking if CONFIG_CGROUP_BPF is enabled"` ?

Reply:
Ah my bad, nope that's fine.